src/staking-v3/components/DiscoverV3.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="wrapper--discover">
+    <!-- Todo: Delete -->
+    <!-- <staking-home /> -->
```

Comment:
@bobo-k2 Can I remove this components?

Reply:
`staking-home` can be removed. Thanks

Reply:
Actually, component is already removed, just remove the commented code